### ISSUE TITLE
Banners query bringing deleted Banners

### ISSUE THREAD

**Issue by victorgrodriguesm7** [vtex-apps/search-graphql#116]
# Banners query bringing deleted Banners

## Expected Behavior
Expected Query to bring only non-deleted banners.

## Current Behavior
Query bring all banners even the deleted ones.

## Context
We created some banners to test and deleted them. Now our customer is trying to create banners that are going to be in the store but the query used in `vtex.search:search-banner` component brings the deleted ones first. We need a way to force delete some banners or some way to filter them.

## Your Environment
* Version used: 0.48.0
* Link to your project: https://netsofas.myvtex.com/--2m?map=largura

**Comment by IvanClementino** [comment]
Had to open a ticket and ask for it to be solved. They removed the "ghost" banners from out website, but no idea about a fix.

**Comment by victorgrodriguesm7** [comment]
Yeah, I received the same reply. They removed the "ghost" banners on the store where I was working but didn't say anything about a fix. :/
